### DISCUSSION TITLE
Why won't my symbol rotate/why won't it add keyframes automatically when I turn it?

### DISCUSSION THREAD

**Discussion by lucaszenodark** [keiwando/toonsquid#512] [Q&A] [answered]
I've been wanting to make a saw that continuously rotates in Toonsquid because I planned on it playing the entire time throughout the animation, _but_  I don't want to spend 5 hours rotating the symbol manually on the timeline so instead I tried to edit the clip but it doesn't want to add keyframes at all on the clip/symbol timeline... I did what other people have told me (and checked for automatic keyframe inserting to make sure it's turned on) so, I don't know if I am missing something crucial, or if it's a bug or something.

_Side Note: I've somewhat skimmed through the github pages and in the Q&A but I haven't really seen anybody with my exact same issue?_

https://github.com/keiwando/toonsquid/assets/155845257/f6515a62-f5f4-40a0-8873-d7fdd5627e2f

**Comment by lucaszenodark** [comment, marked as answer]
Fixed it, credits to a guy on discord who told me how to do it. Basically when you're in the clip (the symbol timeline, not the full animation timeline), press "Create Symbol" again. I tripped out when I saw this because that was something that didn't even cross my mind in the slighting because I was oblivious to that fact and how that worked but that's how you do it to anyone else who ever has the same problem.
![image](https://github.com/keiwando/toonsquid/assets/155845257/de6db198-ed90-4425-a28b-7cd8dffcb169)

So yeah, it didn't spin before but now it does since it creates keyframes through becoming a symbol inside the clip editor. Sounds trippy, but yeah.
![ExampleGifSaw](https://github.com/keiwando/toonsquid/assets/155845257/99a2778c-5b3d-42ff-9516-4a1f303be1e4)

**Comment by andydjames** [comment]
You know that you have to click the Edit Pixels button on the top toolbar to toggle between editing the drawing (pixels) and keyframing in animation mode? Hope that helps too. The Create Symbol is great for creating looping animations but if you need to animate using keyframes more flexibly, then use the Edit Pixels button. The Edit pixels button interacts with the Transform Layer.
